src/sonic_ax_impl/mibs/ieee802_1ab.py

```diff
@@ -112,8 +112,10 @@ def reinit_data(self):
         # establish connection to application database.
         Namespace.connect_all_dbs(self.db_conn, mibs.APPL_DB)
         self.loc_chassis_data = Namespace.dbs_get_all(self.db_conn, mibs.APPL_DB, mibs.LOC_CHASSIS_TABLE)
-        self.loc_chassis_data['lldp_loc_sys_cap_supported'] = parse_sys_capability(self.loc_chassis_data['lldp_loc_sys_cap_supported'])
-        self.loc_chassis_data['lldp_loc_sys_cap_enabled'] = parse_sys_capability(self.loc_chassis_data['lldp_loc_sys_cap_enabled'])
+        if self.loc_chassis_data:
+            self.loc_chassis_data['lldp_loc_sys_cap_supported'] = parse_sys_capability(self.loc_chassis_data['lldp_loc_sys_cap_supported'])
+            self.loc_chassis_data['lldp_loc_sys_cap_enabled'] = parse_sys_capability(self.loc_chassis_data['lldp_loc_sys_cap_enabled'])
```

Comment:
where is the unit test?

Reply:
Existing test case `test_getnextpdu_lldpLocSysCapSupported`

Reply:
the test also passes without this fix?

Reply:
Yes. This is a corner case on DUT with low possibility to repro.
The old impact is syslog error message, and the snmpagent recovered soon after the Redis table is ready. Not a critical bug.